chromatic-protocol/frontend
# Dia/qa 1108

- Common
  - update wording
  - add Testnet Toast for example (can see in storybook) *
    - storybook > Toast > Testnet
    
<img width="348" alt="image" src="https://github.com/chromatic-protocol/frontend/assets/123039938/ff8677fb-4fad-454d-9246-c533f70f7eb7">

- Pool
  - update PoolDetail paragraph bottom - lpTitle, lpName 
  - add metamask buttons (function should be added) *
  - test balance ui (can be updated later)
- Design detail
  - set AddressWithButton width fit-content
  - fix Button style (display flex)
